src/command_buffer.rs

```diff
@@ -30,6 +32,8 @@ use crate::{align, DynamicBundle};
 /// ```
 pub struct CommandBuffer {
     entities: Vec<EntityIndex>,
+    despawn_ent: Vec<Entity>,
+    remove_comps: Vec<Box<dyn Fn(&mut World)>>,
```

Comment:
Probably by wrapping `world.remove::<T>(ent).unwrap()` into another named function? (Is unwrapping really the right thing to do here? Would ignoring the return serve more use case?)

Reply:
> Could this be optimized by storing something like
> 
> ```rust
> Vec<(Entity, fn(&mut World, Entity))>
> ```
> 
> containing elements like
> 
> ```
> (ent, World::remove::<T>)
> ```
> 
> to avoid the boxing overhead as the "shape" of closure is know up front?

Wouldn't that require World to be passed as a parameter on `fn remove` ?